### PR TITLE
feature_release commands:226 add prune parameter to uninstall

### PR DESCRIPTION
**Pull request by daniwe4**
https://github.com/conceptsandtraining/doil/issues/226

### CODE REVIEW

File: setup/install.sh [hunk 1]
```diff
@@ -20,6 +20,9 @@ source ${SCRIPT_DIR}/checks.sh
 source ${SCRIPT_DIR}/log.sh
 source ${SCRIPT_DIR}/system.sh
 
+doil_system_run_composer
+exit
```
Comment (tbongers-cat) [inline setup/install.sh:24]: Is this exit command meant to be here?

File: app/tests/Commands/System/UninstallCommandTest.php [hunk 2]
```diff
@@ -231,8 +231,8 @@ public function test_execute() : void
             ->with("doil")
         ;
 
-        $tester->setInputs(["y"]);
-        $execute_result = $tester->execute([]);
+        $tester->setInputs(["y", "y"]);
+        $execute_result = $tester->execute(["--prune" => true]);
```
Comment (tbongers-cat) [inline app/tests/Commands/System/UninstallCommandTest.php:235]: Maybe add a second test without the prune option?

File: setup/install.sh (hunk 1 above)
Comment (daniwe4) [reply setup/install.sh]: Done!

File: app/tests/Commands/System/UninstallCommandTest.php (hunk 2 above)
Comment (daniwe4) [reply app/tests/Commands/System/UninstallCommandTest.php:235]: Done!

**Review by tbongers-cat — APPROVED** [review]

**Pull request merged**